### PR TITLE
Update HAKRCF405V2

### PR DESCRIPTION
**Pull request by wqz2001**
Configuration for easier flight

### CODE REVIEW

**Comment by haslinghuis** [comment]
Sorry, user settings not allowed in general targets. Each user has it's own set of requirements.

**Comment by KarateBrot** [comment]
The board configuration is the basic configuration to make a board work. It's not meant for storing your personal diffs. You can save your personal diff to a text file and keep it somewhere on your hard drive for your convenience. Or you can make your own personal presets repository.

**Comment by wqz2001** [comment]
> Sorry, user settings not allowed in general targets. Each user has it's own set of requirements.


Hello, I'm an engineer from HAKRC,I’m websom

**Comment by KarateBrot** [comment]
Sorry I wasn't aware. I am reopening, then we can discuss what needs to be done to make this work.

**Comment by wqz2001** [comment]
> 

it's fine.Nice to work with you

**Comment by sugaarK** [comment]
@wqz2001  still need to remove all the user stuff!

**Comment by howels** [comment]
Hi @wqz2001 here's the parts which we would consider changing to remove user configs:

BEFORE (with user config mixed in with defaults)
```
# feature
feature -RX_PARALLEL_PWM
feature RX_SERIAL
feature GPS
feature TELEMETRY
feature OSD

# serial
serial 1 64 115200 57600 0 115200
serial 5 2 115200 57600 0 115200

# beacon
beacon RX_LOST
beacon RX_SET

# aux
aux 0 0 0 1700 2100 0 0
aux 1 1 1 900 1300 0 0
aux 2 2 1 1300 1700 0 0
aux 3 13 2 1700 2100 0 0

# master
set gyro_lpf1_static_hz = 212
set gyro_lpf2_static_hz = 425
set gyro_notch1_hz = 150
set gyro_notch1_cutoff = 100
set gyro_lpf1_dyn_min_hz = 212
set gyro_lpf1_dyn_max_hz = 425
set acc_calibration = -60,13,-2,1
set baro_bustype = I2C
set baro_i2c_device = 1
set serialrx_provider = CRSF
set adc_device = 3
set blackbox_device = SPIFLASH
set dshot_burst = AUTO
set motor_pwm_protocol = DSHOT600
set current_meter = ADC
set battery_meter = ADC
set ibata_scale = 100
set beeper_inversion = ON
set beeper_od = OFF
set beeper_dshot_beacon_tone = 5
set gps_provider = UBLOX
set gps_rescue_min_dth = 50
set gps_rescue_allow_arming_without_fix = ON
set simplified_gyro_filter_multiplier = 85
set osd_vbat_pos = 2371
set osd_rssi_pos = 2116
set osd_tim_2_pos = 2389
set osd_flymode_pos = 2282
set osd_throttle_pos = 2161
set osd_current_pos = 2338
set osd_gps_speed_pos = 2282
set osd_gps_lon_pos = 2129
set osd_gps_sats_pos = 2103
set debug_mode = GYRO_SCALED
set max7456_spi_bus = 2
set flash_spi_bus = 3
set gyro_1_bustype = SPI
set gyro_1_spibus = 1
set gyro_1_sensor_align = CW90
set gyro_1_align_yaw = 900
```

AFTER (no user data, only board-specific data)
```
# master
set baro_bustype = I2C
set baro_i2c_device = 1
set adc_device = 3
set blackbox_device = SPIFLASH
set dshot_burst = AUTO
set ibata_scale = 100
set max7456_spi_bus = 2
set flash_spi_bus = 3
set gyro_1_bustype = SPI
set gyro_1_spibus = 1
set gyro_1_sensor_align = CW90
set gyro_1_align_yaw = 900
```

File: configs/default/HARC-HAKRCF405V2.config [hunk 1]
```diff
@@ -7,6 +7,8 @@
 #define USE_ACCGYRO_BMI270
 #define USE_BARO_BMP280
 #define USE_BARO_SPI_BMP280
+#define USE_BARO_DPS310
+#define USE_BARO_SPI_DPS310
```
Comment (haslinghuis) [inline configs/default/HARC-HAKRCF405V2.config:11]: Remove line 9 and 11 as this board uses I2C for BARO.

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**